src/nodes.lua

```diff
@@ -384,11 +384,14 @@ minetest.override_item("default:snow", {
 		}
 	},
 	leveled = 7,
+	paramtype2 = "leveled",
```

Comment:
What is this used for?
Why do "leveled" nodeboxes work without paramtype2 set to "leveled"?
https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L912